.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -131,8 +131,10 @@ jobs:
       env:
         STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
         STACKROX_NVD_API_CALL_INTERVAL: 6s
-        STACKROX_NVD_ZIP_PATH: nvd.zip
       run: |
```

Comment:
the linter doesn't like this for some reason?

Reply:
Know how to fix it? I didn't change the line it's complaining about.

Reply:
I think it may actually be complaining about the export combined with the $(pwd) and thinking cannot effectively capture the return values, will fix

Reply:
It's happy now: 
![image](https://github.com/user-attachments/assets/5fee13b9-6260-4fa9-817b-041a573ea9db)